macros/activity_schema/activity/generate_activity_yml.sql

```diff
@@ -0,0 +1,74 @@
+-- Get data_types object from model config block
+{% macro get_activity_config(model) -%}
+    {% set relation = ref(model) %}
+    {% for node in graph.nodes.values()
+        | selectattr("resource_type", "equalto", "model")
+        | selectattr("name", "equalto", relation.identifier) %}
+        {% do return(node.config) %}
+    {% endfor %}
+{%- endmacro %}
+
+{% macro generate_activity_yml(activities) %}
+
+{% set yaml_output = [] %}
+
+{% do yaml_output.append('version: 2') %}
+{% do yaml_output.append('models:') %}
+
+-- Loop through each activity
+{% for activity in activities %}
+
+  {% do yaml_output.append('  - name: ' ~ activity) %}
+  {% do yaml_output.append('    columns:') %}
+
+  {% set columns = [
+      {'name': 'activity_id', 'description': 'Unique identifier for the activity.', 'data_type': 'STRING', 'tests': ['unique', 'not_null']},
+      {'name': 'customer_id', 'description': 'Identifier for the customer.', 'data_type': 'STRING'},
+      {'name': 'activity', 'description': 'Type of activity performed by the customer.', 'data_type': 'STRING', 'tests': ['not_null']},
+      {'name': 'ts', 'description': 'Timestamp of when the activity occurred.', 'data_type': 'TIMESTAMP', 'tests': ['not_null']},
+      {'name': 'revenue_impact', 'description': 'Revenue impact of the activity, if applicable.', 'data_type': 'INT64'},
```

Comment:
`revenue_impact` and `link` should be evaluated as optional fields, similar to `anonymous_customer_id`, as they are excluded by default for each activity schema built by this package.

I think dbt only throws a warning when docs contain a reference to a column that doesn't exist (if that), but better to use the configuration information available for more accurate output.

Reply:
Thinking about it more, would it make sense to add all column descriptions including `anonymous_customer_id`, no matter if they are used or not?With the `persist_docs` config (https://docs.getdbt.com/reference/resource-configs/persist_docs) in dbt, the table and column descriptions will be added to the metadata of the database:
![image](https://github.com/bcodell/dbt-aql/assets/44875496/3453ada7-20c4-496f-a57f-77227a76dbea)

Reply:
Ok, I just realized I made a config error when testing this. The columns will be removed from the final table, so it totally makes sense to omit them from the docs output.